_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
Actually you need es6 support because const etc.  So make this your `.eslintrc.js`:

```
module.exports = {
                "env": {
        "browser": true,
        "commonjs": true,
        "es6": true,
        "node": true
    },
    "extends": "google"
};
```